src/components/card/types.ts

```diff
@@ -3,5 +3,5 @@ import { ReactNode } from 'react'
 export interface CardProps {
 	children: ReactNode
 	className?: string
-	elevation?: 'base' | 'mid' | 'high'
+	elevation?: 'base' | 'low' | 'mid' | 'high'
```

Comment:
Added `low` here, as that's what's in the design spec. I think we may want to adjust elsewhere as well, but that felt outside the scope of this PR.